sdk/src/impl/whirlpool-client-impl.ts

```diff
@@ -100,4 +104,87 @@ export class WhirlpoolClientImpl implements WhirlpoolClient {
       account
     );
   }
+
+  public async createPool(
+    whirlpoolsConfig: Address,
+    tokenMintA: Address,
+    tokenMintB: Address,
+    tickSpacing: number,
+    initialTick: number,
+    funder: Address,
+    refresh = false
+  ): Promise<{ pubkey: PublicKey; tx: TransactionBuilder }> {
+    invariant(TickUtil.checkTickInBounds(initialTick), "initialTick is out of bounds.");
+    invariant(
+      TickUtil.isTickInitializable(initialTick, tickSpacing),
+      `initial tick ${initialTick} is not an initializable tick for tick-spacing ${tickSpacing}`
+    );
+
+    [tokenMintA, tokenMintB] = PoolUtil.orderMints(tokenMintA, tokenMintB).map(
+      (addr) => new PublicKey(addr)
+    );
```

Comment:
if the tokenMint order is switched here, we would have to invert the initialTick..

Reply:
true. agree on it failing so that the caller is aware that the price inverted.
As part of this change, can we also
1. provide a util function so callers can easily invert the price/tick
2. add documentation for this method on this failure case. ex. @throws error for inverted case and point them towards the invert price util fn.

To invert, I believe you simply negate the tick value. @philcchen please double check me on this.
> let i = tick
> P(i) = 1.0001^i
> Pb/Pa = 1.0001^i
> 1/ (Pb/Pa) = 1/ (1.0001^i)
> Pa/Pb = 1.0001^-i